geomesa-kafka/geomesa-kafka-09-datastore/pom.xml

```diff
@@ -136,35 +127,22 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-kafka-09-utils</artifactId>
-            <version>${project.version}</version>
-            <type>test-jar</type>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-kafka-utils-common</artifactId>
-            <type>test-jar</type>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
         </dependency>
     </dependencies>
 
     <build>
-        <sourceDirectory>../geomesa-kafka-datastore/src/main/scala</sourceDirectory>
-        <testSourceDirectory>../geomesa-kafka-datastore/src/test/scala</testSourceDirectory>
+        <sourceDirectory>src/main/scala</sourceDirectory>
```

Comment:
Similar comment as in the 08; these directories are probably no longer needed.  Same thing applies to 10, so I'll leave it out there.